Server/src/main/ServerController.java

```diff
@@ -72,7 +64,8 @@ public void handleMessages() {
 			case ADD_USER_BOOK:
 				AddBookToUsersBookListRequest addBookToUsersBookListRequest = gson.fromJson(jsonMessage,
 						AddBookToUsersBookListRequest.class);
-				addBookToUsersList(addBookToUsersBookListRequest.getBookId(), addBookToUsersBookListRequest.getUserId());
+				addBookToUsersList(addBookToUsersBookListRequest.getBookId(),
```

Comment:
change method's signature from "addBookToUsersList(String, String)" to "addBookToUsersBookListRequest(AddBookToUsersBookListRequest)"